aggsender/db/aggsender_db_storage.go

```diff
@@ -564,18 +577,33 @@ func (a *AggSenderSQLStorage) GetNonAcceptedCertificate() (*NonAcceptedCertifica
 	if err := json.Unmarshal([]byte(val), &nonAcceptedCert); err != nil {
 		return nil, fmt.Errorf("failed to unmarshal non-accepted certificate: %w", err)
 	}
+	if nonAcceptedCert.SignedCertificate != "" && nonAcceptedCert.SignedCertificate[0] == '@' {
+		// The content is pointing to a file
+		FullFilename := nonAcceptedCert.SignedCertificate[1:]
+		data, err := os.ReadFile(FullFilename)
+		if err != nil {
+			return nil, fmt.Errorf("getNonAcceptedCertificate: failed to read signed certificate file %s: %w",
+				FullFilename, err)
+		}
+		certHash := common.BytesToHash(keccak256.Hash([]byte(string(data))))
+		if certHash != nonAcceptedCert.CertificateHash {
+			return nil, fmt.Errorf("getNonAcceptedCertificate: certificate hash mismatch: expected %s, got %s (file: %s)",
+				nonAcceptedCert.CertificateHash.String(), certHash.String(), FullFilename)
```

Comment:
I'm proposing the rename here, for the sake of case consistency. Wdyt?
```suggestion
		certificateFilePath := nonAcceptedCert.SignedCertificate[1:]
		data, err := os.ReadFile(certificateFilePath)
		if err != nil {
			return nil, fmt.Errorf("getNonAcceptedCertificate: failed to read signed certificate file %s: %w",
				certificateFilePath, err)
		}
		certHash := common.BytesToHash(keccak256.Hash([]byte(string(data))))
		if certHash != nonAcceptedCert.CertificateHash {
			return nil, fmt.Errorf("getNonAcceptedCertificate: certificate hash mismatch: expected %s, got %s (file: %s)",
				nonAcceptedCert.CertificateHash.String(), certHash.String(), certificateFilePath)
```

Reply:
I understand that the idea is rename `FullFilename` in favour `certificateFilePath`? isn't? I applied that

Reply:
That's right.